pkg/kubelet/kubelet_pods.go

```diff
@@ -1885,6 +1886,13 @@ func (kl *Kubelet) convertToAPIContainerStatuses(pod *v1.Pod, podStatus *kubecon
 			ImageID:      cs.ImageID,
 			ContainerID:  cid,
 		}
+		if !cs.StartedAt.IsZero() && cs.FinishedAt.IsZero() && !hasStartupProbe {
+			// without a startup probe, mark this container started during the time it is running
+			status.Started = pointer.Bool(true)
+		} else {
+			// with a startup probe, prober_manager will set the started field depending on the probe result
+			status.Started = pointer.Bool(false)
+		}
```

Comment:
Do we have a real effect from this, as this field is always overridden by `(*manager).UpdatePodStatus`?

Reply:
> Do we have a real effect from this, as this field is always overridden by `(*manager).UpdatePodStatus`?
> 

Yes because in the meantime we'll have a real value instead of nil... Of course there's a difference between Started and StartedAt because they don't have the same meaning. I think this deterministic value will help us refactor prober manager once we have reached GA.